src/lib/components/private/QDocsSection.svelte

```diff
@@ -13,15 +13,17 @@
   } = $props();
 
   let dialog = $state(false);
+
+  const code = $derived(snippets?.[title].replaceAll(/^  /gm, ""));
 </script>
 
 <div style="margin-bottom:48px">
   <div class="flex justify-between q-mb-md">
     <h5>{title}</h5>
-    {#if snippets}
```

Comment:
Shouldn't `snippets` be removed from all the code here?

Reply:
It has been replaced by `{#if code}`, what do you mean?

Reply:
The variable `snippets` still exists in this component - why not remove it completely?